angular/projects/spark-angular/src/lib/components/sprk-accordion/sprk-accordion.stories.ts

```diff
@@ -28,6 +28,9 @@ ${markdownDocumentationLinkBuilder('accordion')}
 - \`sprk-accordion\` expects you to add \`sprk-accordion-item\`
 components as children. Any other content you add outside of a
 \`sprk-accordion-item\` will render, but may not be styled correctly.
+- If your instance only has accordion item,
```

Comment:
I think we should update this to be "If your instance only has one item, ..." 
Calling it an accordion item feels like maybe they could be confused about what to use.

Reply:
Done!